app-development-wg/charter/README.md

```diff
@@ -0,0 +1,120 @@
+- Code Quality & Standards:
+  - Linters & Formatters
+  - Static Code Analysis Tools
+  - API Standards & Compliance Tools
+- Cloud Native Development:
+  - APIs & SDKs/CLIs for Cloud Services
+  - Standards that developers can adopt
+  - Polyglot Development Support Tools
+
+
+The following diagram illustrates common tools, tasks, and artifacts in developer workflows.
+
+![](imgs/developer-workflows.png)
+
+In addition to uncovering tools, this group should champion best practices for building developer experiences (DevEx) that enhance productivity and minimize cognitive load. This includes promoting tools that integrate seamlessly within existing workflows and empowering teams to design custom interfaces tailored to domain-specific needs.
+
+### Areas In-Scope
+
+A core objective of this group is to identify and promote existing tools, standards, and practices within the CNCF ecosystem. By showcasing how these elements work together, we aim to map how existing tools and standards can enhance and complement practices and tools already familiar to developers and architects.
+
+In addition to identifying best practices, this group should champion open-source tools that promote seamless integration between development tools and adopt standards. These tools empower teams to construct custom interfaces tailored to their specific domains, ultimately reducing cognitive load and boosting developer productivity.
+
+The following list of goals will drive the working group agenda:
+
+- Identify and promote developer productivity tools within the cloud-native ecosystem. Raise awareness and facilitate the adoption of tools supporting and enhancing developer workflows in the cloud-native space.
+- Promote interoperability and standards across tools. Leverage shared APIs and SDKs across the industry to map the development ecosystem, document existing functionalities, and pinpoint areas for improvement.
+- Advocate for developer experience (DevEx) tools that streamline cloud-native development workflows.
+Assist with the organization of AppDeveloperCon, reviewing and proposing topics.
+- Identify how tools contribute to improving developer productivity metrics.
+- Provide early feedback and recommendations for projects and tools seeking CNCF adoption around cloud-native app development. 
+
+
+
+### Areas Out of Scope
+
+These are topics or initiatives that this WG will not target and work on. We might add new items or update this list if we find projects, standards, or initiatives this WG finds interesting to work on in future iterations. 
+- Create new standards.
+- Define how application development should be done.
+- Creation of a new CNCF open-source project.
+- Promote tools that are only useful for a specific stack/programming language.
+- Promote adopting security best practices throughout the development lifecycle, secure coding standards, and vulnerability scanning to mitigate risks and protect applications from potential threats.
```

Comment:
I dont think this should be out of scope.  While TAG Security is largely responsible for this area, this working group should champion and consider secure coding practices (not standards), incorporation of vulnerability remediation, and secure design methodologies as part of their work. This is a supporting function, and will leverage much coordination and existing material from TAG Security but separating secure application development from application development further fractures both domains.  

I would suggest rephrasing this and moving it to in scope as:
```suggestion
- Advocate for secure application development practices as defined by TAG Security alongside any recommendations or material this group produces.
```

Reply:
reading further, under "interface with other TAGs" such a change as proposed only serves to reinforce that collaboration.